### PR TITLE
Refactor/shorter logging in ee

### PR DESCRIPTION
**Pull request by Kwestor**
First commit adds `TRACE` level to logger, second changes long log messages in expression evaluator to use it.

### CODE REVIEW

File: org.scala-ide.sdt.core/src/org/scalaide/core/internal/logging/EclipseLogger.scala [hunk 1]
```diff
@@ -33,47 +33,55 @@ object EclipseLogger extends Logger {
 
   private val lastCrash: AtomicReference[Throwable] = new AtomicReference
 
-  def debug(message: => Any) {
+  def trace(message: => Any): Unit = {
```
Comment (mpociecha) [inline org.scala-ide.sdt.core/src/org/scalaide/core/internal/logging/EclipseLogger.scala]: I would make them one-liners. ;)

Comment (Kwestor) [inline org.scala-ide.sdt.core/src/org/scalaide/core/internal/logging/EclipseLogger.scala]: My too, but I just copy-pasted code below.

**Comment by dragos** [comment]
in the end it doesn't seem to make much of a difference... I guess the bulk comes from somewhere else

**Comment by Kwestor** [comment]
Maybe from project compilation? We have a lot of test projects there and they are compiled before each test class. We have a change that integrates it into single compilation before **all** tests and then clean after, but it needs to be rebased and polished a bit. @romanowski will take care of in some free time.

**Comment by Kwestor** [comment]
@dragos should we merge it anyway?

**Comment by Kwestor** [comment]
Merging, as it does not hurt us to have some logs on `TRACE`.

**Pull request merged**